go.sum

```diff
@@ -1,5 +1,9 @@
 github.com/Financial-Times/go-logger/v2 v2.0.0 h1:amSYqq+6KmOS5E0mg4BtnO+6zLQK9OSWAALxXaaf8wA=
```

Comment:
You can run `go mod tidy` to drop the entries for the old version if `go-logger`.